### PR TITLE
Dbms mongo 4

### PR DESCRIPTION
**Pull request by Vaibhavpratapsingh22**
## Chapter-4 created 🦅 
CRUD OPERATIONS

### CODE REVIEW

File: 5-dbms/9-mongodb/4-crud-operations.md [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+    {"title":"NoSQL Overview"}
+    {"title":"Tutorials Point Overview"}
+
+
+**Note**: _id field is always displayed while executing find() method, if you don't want this field, then you need to set it as 0.
+
+## Update Operations
+The update() method updates the values in the existing document.
+
+### Syntax and Example with command
```
Comment (udeysingh95) [inline 5-dbms/9-mongodb/4-crud-operations.md:96]: This is third level without number

File: 5-dbms/9-mongodb/4-crud-operations.md [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+Below we will see the syntax and example of update operation:
+
+**Syntax**: The basic syntax of update() method is as follows:
+
+    db.COLLECTION_NAME.update(SELECTION_CRITERIA,UPDATED_DATA)
+
+### 1. findOneAndUpdate() method
```
Comment (udeysingh95) [inline 5-dbms/9-mongodb/4-crud-operations.md:103]: This is third level with number. I didn't get - if this is a separate section within the same section or is it a part of "Syntax and example command"

**Review by udeysingh95 — CHANGES_REQUESTED** [review]
There is a whole separate section of practical - I think it would be nice to have it in the theory only. Like when you described them - add the practical screen with some words there only. So its all relevant. Having it all in the end - is like scrolling up and down again and again - to check what was written above and what is it saying here below. 
Also, the readability is pretty low in this PR, somewhere there are numbers, in some places there are not. I also couldn't follow - what is parent and what is child + each section has its body style - which makes it hard to follow till the end. 

Also, README update is missing.

**Comment by Vaibhavpratapsingh22** [comment]
@udeysingh95, I think we should update the README.md at the last.
What u say??

File: 5-dbms/9-mongodb/4-crud-operations.md [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+    			Date_Of_Birth: "1990-02-16",
+    			e_mail: "Fathima_Sheik.123@gmail.com",
+    			phone: "9000054321"
+    		}
+    	]
+    )
+
+### Practical
+Now we will perform practical on our system.\
+**Description**: In the below image we first set the database to employeedb. Then we see that there is collection named emp_details. So we insertOne  document in the collection and then insertMany document in the collection as shown:
```
[the remark this answers is not in the document]
Comment (udeysingh95) [reply 5-dbms/9-mongodb/4-crud-operations.md:68]: As description term is self-understood here and acting only as a brain pillar.

File: 5-dbms/9-mongodb/4-crud-operations.md [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+    			First_Name: "Fathima",
+    			Last_Name: "Sheik",
+    			Date_Of_Birth: "1990-02-16",
+    			e_mail: "Fathima_Sheik.123@gmail.com",
+    			phone: "9000054321"
+    		}
+    	]
+    )
+
+### Practical
```
[the remark this answers is not in the document]
Comment (udeysingh95) [reply 5-dbms/9-mongodb/4-crud-operations.md:66]: This is nice to make it explicit but this practical implementation is expected to be done in this PR - so lets leave out the title.

**Review by udeysingh95 — COMMENTED** [review]
Cool work!

**Review by SaarthakDeshmukh — APPROVED** [review]
The PR is looking perfect. No change from my side. Good Work @Vaibhavpratapsingh22 :rocket:

File: 5-dbms/9-mongodb/4-crud-operations.md [hunk 5]
```diff
@@ -0,0 +1,239 @@
...
+
+**Note**: _id field is always displayed while executing find() method, if you don't want this field, then you need to set it as 0.
+
+Now we will perform practical on our system.\
+ In the image we have first performed the simple find() command without any parameters and then we have used the KEY parameter as discussed above.
+
+<img src="https://user-images.githubusercontent.com/54719422/91700850-18ae8880-eb94-11ea-8f16-625d019dbeb5.png" height="" width="">
+
+
+## Update Operations
```
Comment (udeysingh95) [inline 5-dbms/9-mongodb/4-crud-operations.md:103]:
```suggestion
## 3. Update Operations
```

File: 5-dbms/9-mongodb/4-crud-operations.md [hunk 6]
```diff
@@ -0,0 +1,239 @@
...
+	{ $set: { Age: '00'}}
+    )
+
+Now we will perform practical on our system.\
+In this practical we have performed all the update commands as discussed above. Firstly, we have used UpdateOne command by this we have added a new field in the document named salary. Then we use the find() method to see the updated document. Then we have set the salary of all the employees to 2000 who were having the salary grater then 500. Then we have again used the find method to see the changes.
+
+<img src="https://user-images.githubusercontent.com/54719422/91700862-1b10e280-eb94-11ea-94cf-24c05849bfd1.png" height="" width="">
+
+
+## Delete Operations
```
Comment (udeysingh95) [inline 5-dbms/9-mongodb/4-crud-operations.md:200]:
```suggestion
## 4. Delete Operations
```

**Review by udeysingh95 — CHANGES_REQUESTED** [review]

**Review by udeysingh95 — APPROVED** [review]
